lib/run-script.js

```diff
@@ -125,9 +125,16 @@ function run (pkg, wd, cmd, args, cb) {
   } else {
     if (!pkg.scripts[cmd]) {
       if (cmd === "test") {
-        pkg.scripts.test = "echo \"Error: no test specified\"";
+        pkg.scripts.test = "echo \"Error: no test specified\""
+      } else if (cmd === "env") {
+        log.verbose("run-script using default multiplatform env")
```

Comment:
Ooops... little misunderstanding here -- I'm totally fine with Windows and Unix using different *commands*, I just think we should log which one we're *using*. Does that make more sense?